ts/screens/modal/IdentificationModal.tsx

```diff
@@ -403,48 +404,21 @@ class IdentificationModal extends React.PureComponent<Props, State> {
       </Text>
     ));
 
-  public render() {
-    const { identificationProgressState, isFingerprintEnabled } = this.props;
-
-    if (identificationProgressState.kind !== "started") {
-      return null;
+  /**
+   * Create handlers merging default internal actions (to manage the identification state)
+   * with, if available, custom actions passed as props.
+   */
+  private onIdentificationCancelHandler(
+    identificationCancelData?: IdentificationCancelData
+  ) {
+    if (identificationCancelData) {
+      identificationCancelData.onCancel();
     }
+    this.props.onCancelIdentification();
+  }
```

Comment:
```suggestion
  private onIdentificationCancelHandler(
    identificationCancelData: IdentificationCancelData
  ) {
    identificationCancelData.onCancel();
    this.props.onCancelIdentification();
  }
```
 Atm this method is called only if `identificationCancelData` is defined. Could be worth to refactor it or should be keeped with the optional undefined argument for future use?